_posts/2016-09-14-supercharged-seo-with-drupal-8.md

```diff
@@ -0,0 +1,118 @@
+
+> [There are more mobile internet users than desktop internet users; 52.7% of global internet users access the internet via mobile, and 75.1% of U.S. internet users access the internet via mobile.](https://hostingfacts.com/internet-facts-stats-2016/)
+
+> [4 out of 5 consumers use a Smartphone to shop.](http://www.convinceandconvert.com/mobile/7-mobile-marketing-stats-that-will-blow-your-mind/)
+
+#### 4. Page Speed
+
+In response to the substantial increase in mobile traffic growth, search engines have acknowledged the importance of page speed and the effect it has on user experience (UX) and now give more weight to fast-loading websites.
+
+> [40% of people abandon a website that takes more than 3 seconds to load.](https://blog.kissmetrics.com/loading-time/)
+
+> [a 2-second delay in load time during a transaction resulted in abandonment rates of up to 87%. This is significantly higher than the baseline abandonment rate of 67%.](http://blog.radware.com/applicationdelivery/applicationaccelerationoptimization/2013/05/case-study-page-load-time-conversions/)
+
+Can your business handle the loss in revenue that may occur from slow page load speed?
+
+## Drupal 8 - Built with SEO in Mind
+
+ The base of all our SEO efforts lies within the website’s architecture. There are many website engines and CMS’s to choose from and most of them will claim that they are SEO optimized. Don’t be fooled! No CMS will come search engine optimized out of the box. It may have some features, which, if configured correctly, may bring you some SEO benefits. SEO is not only about code, though it does start there. SEO is also about the continuous efforts of your marketing team. We all know that time = money. The more efficient your marketing team is in performing tasks within your CMS - the more ROI you get!
+
+ A good CMS must provide means for your marketing team to work independently from your development team. Drupal 8 does just that! It provides solid framework that can be tuned to become a powerful marketing-machine.
+
+ Let’s take a look at some of the new features in core that make search engines love Drupal 8.
+
+#### Drupal 8 is Responsive out of the Box
+
+Drupal 8 comes with responsive themes in core. Now both public facing and admin facing themes are responsive and make user experience great on any device.
+
+#### Drupal 8 Page Load is Fast
+
+No! Not out the box! In fact vanilla Drupal 8 is running much more code than vanilla Drupal 7. It runs vendor code like Symfony, which adds some overhead. However, Drupal 8 has a number of improvements that are making up for that overhead:
```

Comment:
> No! Not out the box!

I wonder if this is sending an unnecessarily negative message about pagespeed in D8. I'd delete these two sentences and maybe make the point that although vanilla D8 runs more code than D7 there is great potential for improvement.